Sources/EvervaultInputs/Model/PaymentCardData+Validation.swift

```diff
@@ -18,6 +18,12 @@ internal extension PaymentCardData {
         let formattedNumber = CreditCardFormatter.formatCardNumber(number)
 
         self.card.number = formattedNumber
+        if validator.isValid {
+            self.card.bin = String(number.prefix(8));
```

Comment:
Just a heads up that In the JS SDK,[ it appears that](https://github.com/evervault/evervault-js/blob/ba303f9ffa4154dd3466ab2de0e78dafb4c24275/packages/inputs/src/app/index.ts#L243) American Express BINs are only 6 digits long.